sentences/de/homeassistant_HassCancelTimer.yaml

```
@@ -0,0 +1,12 @@
+language: de
+intents:
+  HassCancelTimer:
+    data:
+      - sentences:
+          - "<timer_cancel>[ den| meinen] Timer"
```

```suggestion
          - "<timer_cancel> [den|meinen] Timer"
```
these additional spaces should not be necessary within the brackets

---

I wasn't aware that double spaces don't cause issues. I too think it looks cleaner like you suggested.

---

just commited without those spaces